group-generators/helpers/data-providers/degenscore/interface-schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "DegenScore",
+  "iconUrl": "",
+  "providerClassName": "DegenScoreProvider",
+  "functions": [
+    {
+      "name": "Get holders of beacon",
+      "functionName": "getBeaconHolders",
```

Comment:
the function `getBeaconHolders` is never defined in you code

Reply:
fixed as well, thanks!